src/plugins/workspace/public/components/workspace_creator/workspace_creator_form.tsx

```diff
@@ -110,6 +114,15 @@ export const WorkspaceCreatorForm = (props: WorkspaceCreatorFormProps) => {
               </EuiPanel>
             </>
           )}
+          <EuiSpacer size="m" />
+          {isDashboardAdmin && (
+            <WorkspacePrivacySettingPanel
```

Comment:
I think we should add permission enabled check here.

Reply:
Added